python-sdk/nuscenes/eval/lidarseg/render.py

```diff
@@ -0,0 +1,327 @@
+import argparse
+import json
+import os
+from typing import Dict, List, Tuple
+
+from matplotlib import pyplot as plt
+from matplotlib import gridspec
+import numpy as np
+from tqdm import tqdm
+
+from nuscenes import NuScenes
+from nuscenes.eval.lidarseg.evaluate import LidarSegEval
+from nuscenes.eval.lidarseg.utils import ConfusionMatrix, LidarsegClassMapper, load_bin_file, LidarSegPointCloud
+from nuscenes.lidarseg.lidarseg_utils import colormap_to_colors, create_lidarseg_legend, get_labels_in_coloring
+
+
+class LidarSegEvalStratified(LidarSegEval):
+    """
+    Extends the LidarSegEval class to provide an evaluation which is stratified by radial distance from the ego lidar.
```

Comment:
Maybe move this line to below?

Reply:
I wrote this here as a docstring for the class, like in NuScenes: https://github.com/nutonomy/nuscenes-devkit/blob/master/python-sdk/nuscenes/nuscenes.py#L38